docs/markdown/50-lifecycle/00-release.md

```diff
@@ -9,6 +9,8 @@ Notes:
 
 Speaker **Romain**
 
+##--##
+
```

Comment:
Remove this and the previous block `notes`, this is one slide and not two

Reply:
Done.